rest/route/distro.go

```diff
@@ -106,7 +106,7 @@ func (h *distroIDChangeSetupHandler) Run(ctx context.Context) gimlet.Responder {
 	}
 
 	d.Setup = h.Setup
-	if err = h.sc.UpdateDistro(d); err != nil {
+	if err = h.sc.UpdateDistro(d, d); err != nil {
```

Comment:
This is an awkward interface. Is there any reason we couldn't just pass the new?

Reply:
because you need to know if something has changed if you need to clear the queue.